### PR TITLE
Replace redundant string formatting

### PR DESCRIPTION
**Pull request by hpohekar**
Replacing all string formatting in f-string format.

### CODE REVIEW

**Review by mkundu1 — APPROVED** [review]

File: src/ansys/fluent/core/launcher/launcher.py [hunk 1]
```diff
@@ -207,19 +207,11 @@ def _build_fluent_launch_args_string(**kwargs) -> str:
...
-                        allowed_values,
+                        f"Default value {argval} is chosen for {k} as the passed, value {old_argval} is outside, allowed values {allowed_values}."
                     )
                 else:
                     logger.warning(
-                        "%s = %s is discarded as it is outside " "allowed values %s.",
-                        k,
-                        argval,
-                        allowed_values,
+                        f"{k} = {argval} is discarded as it is outside, allowed values {allowed_values}"
```
Comment (raph-luc) [inline src/ansys/fluent/core/launcher/launcher.py:214]:
```suggestion
                        f"{k} = {argval} is discarded as it is not an allowed value. Allowed values: {allowed_values}"
```

**Review by raph-luc — APPROVED** [review]

File: src/ansys/fluent/core/launcher/launcher.py [hunk 2]
```diff
@@ -207,19 +207,11 @@ def _build_fluent_launch_args_string(**kwargs) -> str:
                     old_argval = argval
                     argval = default
                     logger.warning(
-                        "Default value %s is chosen for %s as the passed "
-                        "value  %s is outside allowed values %s.",
-                        argval,
-                        k,
-                        old_argval,
-                        allowed_values,
+                        f"Default value {argval} is chosen for {k} as the passed, value {old_argval} is outside, allowed values {allowed_values}."
```
[the remark this answers is not in the document]
Comment (hpohekar) [reply src/ansys/fluent/core/launcher/launcher.py]: Done. Thank you.

File: src/ansys/fluent/core/launcher/launcher.py (hunk 1 above)
Comment (hpohekar) [reply src/ansys/fluent/core/launcher/launcher.py]: Done. Thank you.

**Comment by hpohekar** [comment]
@seanpearsonuk It will be good if you share your views. Thank you.

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**